rclpy/test/test_node.py

```diff
@@ -612,6 +612,60 @@ def test_node_get_undeclared_parameter_or(self):
         self.assertEqual(result.name, 'foo')
         self.assertEqual(result.value, 152)
 
+    def test_node_get_parameters_by_prefix(self):
+        parameters = [
+            ('foo.foo', 43),
+            ('foo.bar', 'hello'),
+            ('foo.baz', 2.41),
+            ('bar.foo', 1),
```

Comment:
I think it would be better to either change the name of `foo.bar`, or the prefix `bar` in this code.  At first glance it is confusing whether we are checking the specific key or the prefix in the code below.

Reply:
Thanks for the comment; addressed in https://github.com/ros2/rclpy/pull/386/commits/d031737d6d658ce15a8f065ad1822e6a206af458.

Reply:
Thanks!